packages/schematics/angular/migrations/update-8/update-lazy-module-paths.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';  // tslint:disable-line:no-implicit-dependencies
+import {
+  TslintFixTask,
+} from '@angular-devkit/schematics/tasks';  // tslint:disable-line:no-implicit-dependencies
+import * as path from 'path';
+
+export const updateLazyModulePaths = (): Rule => {
+  return (_: Tree, context: SchematicContext) => {
+    context.addTask(new TslintFixTask({
+      rulesDirectory: path.join(__dirname, 'rules'),
+      rules: {
+        'no-lazy-module-paths': [true],
+      },
+    }, {
+      includes: '*.ts',
```

Comment:
Shouldn't this be `**/*.ts`?

Reply:
Damn, I hoped this was going to get the tests passing 😞